tools/release/release_deb_monorepo.py

```diff
@@ -221,7 +221,13 @@ def bump(self):
                 else:
                     # bump to jump to rc1
                     run(['bumpversion', '--current-version',
-                         '1.99.0',  # FIXME last_tag,
+                         # FIXME Using 1.99.0 here ensures all packages will
```

Comment:
I can't say I follow your thought there fully, but sounds like we have a follow-up task to consistently version the packages so we don't have differing major (or indeed minor) versions across packages. Presumably this also conflicts with giving per package control when releasing too, so needs some thought.